org.adempiere.base/src/org/adempiere/base/DefaultCreditManagerFactory.java

```diff
@@ -0,0 +1,48 @@
+/******************************************************************************
+ * Copyright (C) 2016 Logilite Technologies LLP								  *
+ * This program is free software; you can redistribute it and/or modify it    *
+ * under the terms version 2 of the GNU General Public License as published   *
+ * by the Free Software Foundation. This program is distributed in the hope   *
+ * that it will be useful, but WITHOUT ANY WARRANTY; without even the implied *
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.           *
+ * See the GNU General Public License for more details.                       *
+ * You should have received a copy of the GNU General Public License along    *
+ * with this program; if not, write to the Free Software Foundation, Inc.,    *
+ * 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA.                     *
+ *****************************************************************************/
+package org.adempiere.base;
+
+import org.compiere.model.MInOut;
+import org.compiere.model.MInvoice;
+import org.compiere.model.MOrder;
+import org.compiere.model.MPayment;
+import org.compiere.model.PO;
+import org.compiere.model.credit.CreditManagerInOut;
+import org.compiere.model.credit.CreditManagerInvoice;
+import org.compiere.model.credit.CreditManagerOrder;
+import org.compiere.model.credit.CreditManagerPayment;
+
+/**
+ * Default Credit Manager Factory
+ * 
+ * @author Logilite Technologies
+ * @since  June 25, 2023
+ */
```

Comment:
this is not a must but it is preferable to use component annotation instead of manually created xml definition (`@Component(immediate=true, service=ICreditManagerFactory.class)`)

Reply:
This changes done and tested.